docgen/src/examples/tourism/App.js

```diff
@@ -234,11 +239,15 @@ const RoomType = RefinementList.connect(({items, refine, selectedItems}) => {
 });
 
 function Price() {
+  const theme = _.entries(Range.defaultTheme).reduce((acc, i) => {
```

Comment:
Saying that the default theme as a key root bind to classname A and the user theme as a key root bind to classname B.
If you perform an erase merge root will be bind to classname B. 
If you perform a combined merge, root will be bind to classname A and B. 

Maybe not the better name though :)